storage/HashTable.hpp

```diff
@@ -1429,6 +1492,11 @@ HashTablePutResult HashTable<ValueT, resizable, serializable, force_key_copy, al
                                        variable_size,
                                        (*functor)(*accessor),
                                        using_prealloc ? &prealloc_state : nullptr);
+            // Insert into bloom filter, if enabled.
+            if (has_build_side_bloom_filter_) {
+              this->build_bloom_filter_->insert(static_cast<const std::uint8_t *>(key.getDataPtr()),
```

Comment:
I am not completely sure if this code will be thread safe. I think we should have another round of discussion to discuss the thread-safety of using bloom filters inside the hash table.